google-cloud-clients/google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/testing/FakeStorageRpc.java

```diff
@@ -173,7 +177,7 @@ public StorageObject create(StorageObject object, InputStream content, Map<Optio
     // again).
     // The type cast seems to be necessary to help Java's typesystem remember that collections are
     // iterable.
-    return Tuple.of(null, (Iterable<StorageObject>) values);
+    return Tuple.of(pageToken, (Iterable<StorageObject>) values);
```

Comment:
I'm not sure if I understand replacing `null` with `pageToken` could explain this bit?

In addition, please add a unit test to exercise this change.

Reply:
We have the issue [#4152](https://github.com/googleapis/google-cloud-java/issues/4152) saying that there an inconsistency of LocalStorageHelper and real storage. 

The real implementation of StorageRpc list method in HttpStorageRpc class https://github.com/googleapis/google-cloud-java/blob/06b0ed50e31e9bef72ffac1788e0ec9ff9b9b6dd/google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java#L329 returns 

return Tuple.<String, Iterable<Bucket>>of(buckets.getNextPageToken(), buckets.getItems());

that allows have multiple pages, 
but the the list method in FakeStorageRpc  https://github.com/googleapis/google-cloud-java/blob/06b0ed50e31e9bef72ffac1788e0ec9ff9b9b6dd/google-cloud-clients/google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/testing/FakeStorageRpc.java#L176
 always returns null, that means only one page.

The PR above allows removing the inconsistance. 
Regarding a unit test, I'm not sure that a new unit test may cover that case, I would appreciate your input.

Reply:
Thanks for clarifying and ignore my request for a unit test.